arch/arm/src/stm32h7/stm32h7x3xx_rcc.c

```diff
@@ -144,10 +148,6 @@ static inline void rcc_reset(void)
               RCC_CR_PLL2ON | RCC_CR_PLL3ON |
               RCC_CR_HSIDIV_MASK);
 
-  /* Set HSI predivider to default (4, 16MHz) */
```

Comment:
@jlaitine do you see any issue with the change of order of operation here?

Reply:
No problem. I don't have access to that (proprietary) HW any longer, and I don't remember any more why it was done like that. Anyhow, now the original code looks wrong and the patch looks correct to me.